itest/itest.sh

```diff
@@ -63,18 +64,32 @@ function wait_for_tame_output {
 function main {
   is_port_used
   if [ "$DEBUG" = "true" ]; then set -x; fi
-  echo "Building itest image"
-  # Build the base image
-  docker build -t pidtree-itest-base .
-  # Run the setup.sh install steps in the image so we don't hit timeouts
-  docker build -t pidtree-itest itest/
   mkfifo $FIFO_NAME
-  echo "Launching itest-container $CONTAINER_NAME"
-  docker run --name $CONTAINER_NAME -d\
-      --rm --privileged --cap-add sys_admin --pid host \
-      -v $(git rev-parse --show-toplevel)/itest/example_config.yml:/work/config.yml \
-      -v $(git rev-parse --show-toplevel)/$FIFO_NAME:/work/outfile \
-      pidtree-itest -c /work/config.yml -f /work/outfile
+  if [[ "$@" = "" ]]; then
```

Comment:
If i type `make itest_xenial` this `$@` variable becomes "--xenial", so neither a docker or debian is created and the tests fail. Perhaps we should catch that situation and give the user back an error about what make itest_ targets are supported

Reply:
++, fixed